derive/src/pyclass.rs

```diff
@@ -317,18 +317,25 @@ where
         let item_meta = MethodItemMeta::from_attr(ident.clone(), &item_attr)?;
 
         let py_name = item_meta.method_name()?;
-        let build_func = Ident::new(&format!("build_{}", &self.method_type), args.item.span());
         let tokens = {
             let doc = args.attrs.doc().map_or_else(
                 TokenStream::new,
                 |doc| quote!(.with_doc(#doc.to_owned(), ctx)),
             );
+            let build_func = match self.method_type.as_str() {
+                "method" => quote!(.build_method(ctx, class)),
```

Comment:
My main motivation for this change was the fact that "Find usages" and `grep` failed to find this `.build_method` call site.

Which resulted in 1000s of errors in the project. It was quite hard to find the reason 🙂